www/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
Delete this as it is covered by the kubernetes license (and if it's not this all should be in third_party)